packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -501,6 +501,21 @@ const schema: SchemaType<DbTag> = {
     foreignKey: "Users",
     optional: true,
   },
+  subforumIntroPostId: {
+    ...foreignKeyField({
+      idFieldName: "subforumIntroPostId",
+      resolverName: "subforumIntroPost",
+      collectionName: "Posts",
+      type: "Post",
+    }),
+    optional: true,
+    viewableBy: ['guests'],
+    editableBy: [userIsSubforumModerator, 'sunshineRegiment', 'admins'],
```

Comment:
Should we expose this to subforum mods? I feel like it's something we'd want to set internally for now. Plus they can't always click on the "Edit" button anyway - they would need at least 10 karma.